src/sipmanalyze/estimate.py

```diff
@@ -0,0 +1,121 @@
+"""
+
+estimate.py
+
+Functions to help with estimation of low-light parameters.
+
+"""
+
+import sys
+sys.path.insert(0, '../../sipmpdf')
+sys.path.insert(0, '../')
+
+import matplotlib.pyplot as plt
+import src.sipmanalyze.formats as forms
+import awkward as ak
+import numpy as np
+
+from scipy.signal import find_peaks
+import scipy.integrate as integrate
+from scipy.optimize import curve_fit
+
+
+def import_data(file_location,plot=True):
+  container = forms.waveform.waveform_container.from_root(file_location)
+  tint = container.settings.timeintervals
+  vadc = container.settings.adc_val
+  area = ak.sum(container.waveforms[:, 5:30], axis=-1) * tint * vadc * -1
+  amin, amax = ak.min(area), ak.max(area)
+  nbins = int((amax - amin) / tint / vadc / 4)
+
+  n, bins, patches= plt.hist(x=area, bins=nbins, range=(amin, amax), label='r')
+  total_elements=sum(n)
+  bin_width=(amax-amin)/nbins
+  
+  if plot:
+    plt.title("Lowlight Data")
+    plt.xlabel('Readout value mv-ns')
+    plt.ylabel('Number of events')
+    plt.legend()
+   
+  results={"raw_data":area,"amax":amax,"amin":amin,"nbins":nbins,"bin_width":bin_width, "n":n,}
+  return results
+
+
+
+def get_peaks(import_data_results, plot=True, min_gain_guess=80, min_ratio_guess=1/20):  
+
```

Comment:
Add a short description as to what computation routine is used (like link to scipy.signal.find_peaks for readers to go to), and needs to be provided by the users.